app/src/pages/inside/launchesPage/modals/launchImportModal/launcIcon/iconStateIndicators/iconStateIndicators.scss

```diff
@@ -0,0 +1,19 @@
+.indicator-icon {
+  height: 16px;
+  width: 16px;
+  margin: 0 auto;
+}
+
+.indicator-default {
+  text-transform: uppercase;
+  font-size: 15px;
+  font-family: $FONT-REGULAR, sans-serif;
+  color: $COLOR--gray-91;
+  font-weight: bold;
```

Comment:
Do not use font-weight property in our project. If you need bolde font please use $FONT-SEMIBOLD variable.